packages/jbrowse-web/src/plugins/LinearGenomeView/components/Rubberband.js

```diff
@@ -0,0 +1,141 @@
+import React, { Component } from 'react'
+import { PropTypes } from 'prop-types'
+import { PropTypes as MobxPropTypes } from 'mobx-react'
+
+function getBp(px, model) {
+  const regions = model.displayedRegions
+  const bp = (model.offsetPx + px - model.controlsWidth) * model.bpPerPx + 1
+  let bpSoFar = 0
+  for (let i = 0; i < regions.length; i += 1) {
+    const region = regions[i]
+    if (region.end - region.start + bpSoFar > bp && bpSoFar <= bp) {
+      return Object.assign({}, region, { offset: bp - bpSoFar, index: i })
+    }
+    bpSoFar += region.end - region.start
+  }
+  return undefined
+}
+function navigateToLocation(start, end, model) {
+  // find locations in the modellist
+  let bpSoFar = 0
+  if (start.index === end.index) {
+    bpSoFar += end.offset - start.offset
+  } else {
+    bpSoFar += start.end - start.offset
+    bpSoFar += end.offset
+    if (end.index - start.index > 2) {
+      for (let i = start.index + 1; i < end.index - 1; i += 1) {
+        bpSoFar +=
+          model.displayedRegions[i].end - model.displayedRegions[i].start
+      }
+    }
+  }
+  let bpToStart = 0
+  for (let i = 0; i < model.displayedRegions.length; i += 1) {
+    const region = model.displayedRegions[i]
+    if (start.index === i) {
+      bpToStart += start.offset
+      break
+    } else {
+      bpToStart += region.end - region.start
+    }
+  }
+  const bpPerPx = bpSoFar / model.width
+  console.log(bpToStart, bpPerPx)
+  const offsetPx = bpToStart / bpPerPx
+  model.setNewView(bpPerPx, offsetPx)
+  return 0
+}
+
+class Rubberband extends Component {
+  static defaultProps = {
+    style: {},
+    children: undefined,
+  }
+
+  static propTypes = {
+    model: MobxPropTypes.objectOrObservableObject.isRequired,
+    children: PropTypes.node,
+    style: PropTypes.objectOf(PropTypes.any),
+  }
+
+  state = {}
+
+  onMouseDown = ({ clientX }) => {
```

Comment:
Here's an example of event.preventDefault()

```suggestion
  onMouseDown = event => {
  event.preventDefault()
  const { clientX } = event
```

Reply:
I went ahead and made this change. I guess my feeling was that text selection over these regions didn't make sense but perhaps that's not always true